zng/docs/spec.md

```diff
@@ -110,6 +110,16 @@ Note that the value encoding need not refer to the field names and types as that
 completely captured by the type ID.  Values merely encode the value
 information consistent with the referenced type ID.
 
+Since type definitions are carried in-band in a ZNG stream,
+the entire stream must generally be processed in-order to maintain
+accurate and complete information on types.
+To facilitate random access into large stored ZNG streams, a stream
+may also be optionally organized into a sequence of "frames",
+each of which has an independent type context and can thus be
+processed correctly without first processing the entire preceding stream.
+This benefit comes at the cost of some additional overhead --
+the space consumed by frame boundary markers and repeated type definitions.
+
```

Comment:
Are we adding framing to the ZNG text format? I see no reason we should, and if we don't, then framing isn't part of the ZNG data model, and this paragraph belongs in the next section.